packages/feedback/components/feedback/__stories__/feedback.stories.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * More on writing stories with args: https://storybook.js.org/docs/web-components/writing-stories/args
+ *
+ * @type {{args: {label: string}, render: (function(*): TemplateResult<1>)}}
+ */
+export const Default = {
+  argTypes: defaultControls,
+  args: {
+    ...defaultArgs,
+    'feedback-list': [
+      {
+        feedback_id: '386a4a10-49a4-47c0-9ef0-2eeb8280f97a',
+        generation_id: '2289fec2-93a5-4274-b62b-c905bffcdc3c',
+        start_index: 4,
+        end_index: 18,
+        selected_text: 'standard chunk',
+        suggested_text: 'standard chunk',
+        feedback_type: ['SOCIAL_BIAS'],
+        comment: 'standard part',
+      },
+      {
+        feedback_id: 'd25da065-b9f4-4542-8b9d-9bfb4496e3d7',
+        generation_id: '2289fec2-93a5-4274-b62b-c905bffcdc3c',
+        start_index: 22,
+        end_index: 54,
+        selected_text: 'Lorem Ipsum used since the 1500s',
+        suggested_text: 'Lorem Ipsum used since the 1500s',
+        feedback_type: ['HAP'],
+        comment: 'testing',
+      },
+      {
+        feedback_id: '8d68871b-7f00-40ca-9828-a36689fb04a4',
+        generation_id: '2289fec2-93a5-4274-b62b-c905bffcdc3c',
+        start_index: 223,
+        end_index: 245,
+        selected_text: 'accompanied by English',
+        suggested_text: 'accompanied by English',
+        feedback_type: ['HAP', 'PII'],
+        comment: '',
+      },
+      {
+        feedback_id: '43d906de-9333-448b-97da-07819381b6a4',
+        generation_id: '2289fec2-93a5-4274-b62b-c905bffcdc3c',
+        start_index: 166,
+        end_index: 192,
+        selected_text: 'Cicero are also reproduced',
+        suggested_text: 'Cicero are also reproduced',
+        feedback_type: ['HAP'],
+        comment: '',
+      },
+    ],
+  },
+
+  parameters: {
+    disable: true,
+    docs: { inlineStories: false, iframeHeight: 1200 },
+    controls: {
+      expanded: true,
+    },
+  },
+
+  /**
+   * Renders the template for Storybook
+   * @param {object} args Storybook arguments
+   * @returns {TemplateResult<1>}
+   */
+  render: (args) =>
+    html`
+      <!-- <h2>${args.input}</h2> -->
+      <c4ai-feedback
+        @on-feedback-delete=${args['on-feedback-delete']}
+        @on-feedback-save=${args['on-feedback-save']}
+        model=${args.model}
+        input=${args.input}
+        output=${args.output}
+        generation-id=${args['generation-id']}
+        .feedbackList=${args['feedback-list']}
+        icon=${args.icon}>
+        ${Flag24({ slot: 'icon' })} ${args.output}
+      </c4ai-feedback>
+    `,
+};
+
+export const Playground = {
+  argTypes: defaultControls,
+  args: {
+    ...defaultArgs,
+    /**
+     *
+     * @param {CustomEvent} event custom event fires when new message is generated
+     */ 'on-feedback-delete': (event) => console.log(event.detail),
+    /**
+     *
+     * @param {CustomEvent} event custom event fires when feedback saved
+     */
+    'on-feedback-save': (event) => console.log(event.detail),
```

Comment:
aren't the custom events already part of `defaultArgs`?

Reply:
In defaultArgs I am logging whole event. But here i am logging the actual data. If there is any issue i can remove that.